app/Metadata/LycheeVersion.php

```diff
@@ -29,6 +43,15 @@ private function fetchReleaseInfo()
 		return !file_exists(base_path('.git'));
 	}
 
+	/**
+	 * Returns true if we are using the release channel
+	 * Returns false if we are using the git channel.
+	 */
+	private function fetchComposerInfo()
+	{
+		return file_exists(base_path('vendor/bin/phpunit'));
```

Comment:
Could this just be on line 34? Does it need its own function?

Reply:
It is just for caching data. :x Sure could be refactored. Not a strong opinion on that one.

Reply:
So long as there's a reason, I'm happy.